### DISCUSSION TITLE
Student Developer Pack Application Wrongly Rejected For Ambiguous Reason

### DISCUSSION THREAD

**Discussion by CystroNofer** [community/community#137893] [GitHub Education]
### Hi everyone,

Hi, I'm currently enrolled in a university in the US, and would like to apply for the Student Developer Pack. I have:
- Set my profile name and the name on my billing information to my real name
- Set my billing address to my dorm (which is on campus)
- Tried applying with a government-issued enrollment document and a school-issued transcript, both with my name and a date present.

However, my applications kept being rejected, and the reason for the rejection simply said "Please use your device camera to submit your academic affiliation documents. If you’re having trouble using your device camera, please allow camera access for the application website on your browser..." Which does not make much sense since my photos were successfully uploaded and processed.

I also tried to open a support ticket, but a ticket regarding this issue was not allowed ("Individual manual reviews and status updates are no longer provided.")

How can I have my documents reviewed by a real person, or how can I have my documents successfully recognized? Any help would be greatly appreciated.

Nofer

**Comment by Aritz-Garcia** [comment]
Hi,

I was finally able to fix it by completing the process from my phone. The only downside is that I had to take the photo with the front camera, as it wouldn’t allow me to use the rear camera at all.

I hope this helps. 😊

**Reply by CystroNofer** [reply]
Hi there,

Thx for the reply, but that also didn't work for me...

Thx anyway

**Comment by aivo9** [comment]
Captain here!

I had the same issue, but I just got approved.

Got rejected multiple times. Time after time. Made few perceptions about how it works, and finally succeeded!

1. Put everything in order in your account settings. Just as the instructions advice you to do. Name, emails, bios, 2 phase authentication, etc.
2. Try to photograph different kind of official school documents with a bad webcam and get rejected multiple times over and over again.
3. Get frustrated.
4. Realize that there is no human approving your entries but artificial intelligence instead "reading" words from the pictures you send.
5. Improvise, adapt, overcome.
6. Open the notebook of your mobile phone and type:

[Your University of Fancy Science]

Student: [insert your name here]
School: [insert your school name (same as in the title) here]
Education: [insert the name of your education here] (I had to add ",Virtual learning" to it cuz I'm not on campus right now, and it got mad)
Study right: [insert the day when you got your study right and the day it ends (in US format e.g. 1 January 2024 - 31 December 2028)]

7. So basically everything is true and you are not lying, but you just made the proof yourself on your mobile phone's notepad.
8. Take a screenshot. Edit the picture with your mobile phone and cut off all the excess space so that the text looks as big as possible in the screen.
9. Open the edited picture on your mobile phone and take the phone so close to your laptop's webcam so that all the words are clearly visible in the picture.
10. Take the photo and check that all the information is visible in the photo.
11. Submit your application.
12. Get approved few minutes later.
13. Not your fault the system of approval is so horrible and broken so feel no shame about cutting the corners.
14. Return the favor and go help others.
15. Enjoy!

Welcome to the community!

*flies away*

**Reply by CFokstuen** [reply]
This is the answer.
